agent/acs/model/ecsacs/api.go

```diff
@@ -236,6 +236,10 @@ type Container struct {
 
 	Secrets []*Secret `locationName:"secrets" type:"list"`
 
+	StartTimeout *int64 `locationName:"startTimeout" type:"integer"`
```

Comment:
Can we make sure ECS service treats it as int64 as well?

Reply:
we're treating this field as we do with cpu/mem